src/sentry/eventstore/processing/default.py

```diff
@@ -1,14 +1,12 @@
 from sentry.cache import default_cache
 from sentry.utils.kvstore.cache import CacheKVStorage
 
-from .base import BaseEventProcessingStore
+from .base import EventProcessingStore
 
 
-class DefaultEventProcessingStore(BaseEventProcessingStore):
+def DefaultEventProcessingStore() -> EventProcessingStore:
     """
-    Default implementation of processing store which uses the `default_cache`
-    as backend.
+    Creates an instance of the processing store which uses the
+    ``default_cache`` as its backend.
     """
-
-    def __init__(self, **options) -> None:
-        super().__init__(inner=CacheKVStorage(default_cache), **options)
```

Comment:
Moved these to factory methods since there is no reason for them to be subclasses (they should not override the behavior of the superclass, their interface is defined enough to be encapsulated on its own for the purpose of substitutability) and this reduces potential confusion with what arguments are forwarded where. From the caller's point of view (i.e. existing configuration), nothing changes.